java/android/nnstreamer/src/main/jni/Android-nnstreamer.mk

```diff
@@ -39,13 +39,48 @@ NNSTREAMER_SRC_FILES += \
     $(NNSTREAMER_JOIN_SRCS)
 endif
 
+# TODO: Add nnsquery prebuilt-lib and enable mqtt-hybrid
+ifeq ($(ENABLE_MQTT), true)
```

Comment:
Let's move these changes to line 40.
(When building library single-shot only, this is unnecessary.)

Reply:
Thanks. I moved the part. PTAL